src/pages/Profile.js

```diff
@@ -117,24 +121,26 @@ const NihLink = ({ nihToken }) => {
 
   const renderStatus = () => {
     const { linkedNihUsername, linkExpireTime, datasetPermissions } = nihStatus
-    return h(Fragment, [
-      !linkedNihUsername && h(ShibbolethLink, ['Log in to NIH to link your account']),
-      !!linkedNihUsername && div({ style: { display: 'flex', flexDirection: 'column', width: '33rem' } }, [
-        div({ style: { display: 'flex' } }, [
-          div({ style: { flex: 1 } }, ['Username:']),
-          div({ style: { flex: 2 } }, [linkedNihUsername])
-        ]),
-        div({ style: { display: 'flex' } }, [
-          div({ style: { flex: 1 } }, ['Link Expiration:']),
-          div({ style: { flex: 2 } }, [
-            div([Utils.makeCompleteDate(linkExpireTime * 1000)]),
-            div([h(ShibbolethLink, ['Log in to NIH to re-link your account'])])
-          ])
-        ]),
-        _.flow(
-          _.sortBy('name'),
-          _.map(renderDatasetAuthStatus)
-        )(datasetPermissions)
+    return div({ style: styles.identityLine }, [
+      h(Fragment, [
```

Comment:
This `Fragment` isn't necessary, since there's a `div` that can contain the children.